app/models/collection_draft.rb

```diff
@@ -94,8 +94,11 @@ def update_draft(params, editing_user_id)
 
       # Convert {'0' => {'id' => '123'}} to [{'id' => '123'}]
       params = convert_to_arrays(params.clone)
-      # Convert parameter keys to CamelCase for UMM
-      json_params = params.to_camel_keys
+
+      # Convert parameter keys to CamelCase for UMM;
+      # amend_license_url_key changes the 'LicenseURL' key to 'LicenseUrl'; the .to_camel_keys method has been patched to return 'LicenseURL'
+      # when 'license_url' is passed which is only valid for Service and Tool metadata; see awrence_patch.rb for details
+      json_params = amend_license_url_key(params.to_camel_keys)
```

Comment:
I think we should leave this line as it originally was, and call the new method in the next line (and move the comments with it) so it doesn't change the original operation but also is clear that it is another step. I know in ruby we often want to combine things into one line, but this is a fairly complicated method that I think it is beneficial to make each operation more discrete.